packages/config/scripts/tokens/add/index.ts

```diff
@@ -10,14 +10,18 @@ import { getCanonicalTokens } from '../../../src'
 import { getEnv } from '../../checkVerifiedContracts/utils'
 import { getTokenInfo } from './getTokenInfo'
 
+const API_KEY_ENV_FALLBACK = '_FALLBACK'
+
 async function main() {
   const [address, category] = handleCLIParameters()
   if (!address) {
     return
   }
 
   const http = new HttpClient()
-  const coingeckoClient = new CoingeckoClient(http, getEnv('COINGECKO_API_KEY'))
+  const apiKeyEnv = getEnv('COINGECKO_API_KEY', API_KEY_ENV_FALLBACK)
```

Comment:
wouldn't it be better to just do `process.env.COINGECKO_API_KEY`? it can resolve to undefined